client/src/app/components/stock-detail-header/stock-detail-header.component.ts

```diff
@@ -25,6 +25,9 @@ export class StockDetailHeaderComponent {
   get stockValue(): string {
     return this.stockInfo ? this.stockInfo.stockValue.toFixed(2) : '';
   }
+  get stockIndustry(): string {
+    return this.stockInfo ? this.stockInfo.industry : '';
+  }
```

Comment:
A test should be written for this function. Yes I know it's small, but it has logic so we must test it